test/test.js

```diff
@@ -200,3 +200,60 @@ describe('Tests email sender module', () => {
     expect(result.message).to.be.a('string');
   });
 });
+
+describe('Test for user Login', () => {
```

Comment:
Kindly add this to a users test file instead of having all the tests in one large file.